weaviate/templates/weaviateServiceMonitor.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.env.PROMETHEUS_MONITORING_ENABLED .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: weaviate
+  labels:
+    name: weaviate
+    app: weaviate
+    app.kubernetes.io/name: weaviate
+    app.kubernetes.io/managed-by: helm
+spec:
+  jobLabel: weaviate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: weaviate
+      app.kubernetes.io/managed-by: helm
```

Comment:
Must equal with the [service's labels](https://github.com/weaviate/weaviate-helm/blob/master/weaviate/templates/weaviateService.yaml#L6-L7)